src/Pages/Repositories/PopularRepositoriesTable/PopularRepositoriesTable.tsx

```diff
@@ -556,7 +562,7 @@ const PopularRepositoriesTable = () => {
             <Thead>
               <Tr>
                 <Hide hide={!rbac?.repoWrite}>
-                  <Th
+                  <Th screenReaderText='empty'
```

Comment:
although this works a better method to remove the console error is to add an aria-label for the screen reader to read as seen [here](https://github.com/xbhouse/content-sources-frontend/blob/758d1f0aa4abd51573068c35d24343f5222fae66/src/Pages/Repositories/ContentListTable/ContentListTable.tsx#L534-L535)